File: javakatas/src/FizzBuzz.java
/**
 * Created by erinmoore on 3/15/16.
 */
public class FizzBuzz {
    public String fizzbuzzer (int input){
        String return_value = "";
        if (input%3 == 0){
            return_value += "Fizz";
        }

        if (input%5 == 0){
            return_value += "Buzz";
        }
        if (return_value == ""){
        return_value = Integer.toString(input);
        }
        return return_value;
    }
}

File: pythonkatas/fizzbuzz/fizzbuzz.py
import sys

class Fizzbuzz():

    def fizzbuzzer(self, input_int):
        input = input_int
        returnitem = ""
        if input % 5 == 0:
            returnitem += "Fizz"
        if input % 3 == 0:
            returnitem += "Buzz"
        if returnitem == "":
            returnitem = input
        return returnitem


if __name__ == '__main__':
    main = Fizzbuzz
    fizzbuzz = Fizzbuzz()
    print (fizzbuzz.fizzbuzzer(int(sys.argv[1])))
    sys.exit(0)
